src/networks.ts

```diff
@@ -418,4 +418,26 @@ export const networkInfo = new Map<number, NetworkInfo>([
       stagingBaseAPI: 'https://transaction-testnet.staging.safe.kava.io',
     },
   ],
+  [
+    660279,
+    {
+      chainID: 660279,
+      name: 'Xai',
+      shortName: 'xai',
+      currencySymbol: 'XAI',
+      baseAPI: 'https://transaction.safe.xai.games',
+      stagingBaseAPI: 'https://transaction-testnet.safe.xai.games',
+    },
+  ],
+  [
+    37714555429,
+    {
+      chainID: 37714555429,
+      name: 'Xai Testnet Sepolia',
+      shortName: 'Xai-testnet-sepolia',
+      currencySymbol: 'sXAI',
+      baseAPI: 'https://transaction.staging.safe.xai.game',
```

Comment:
```suggestion
      baseAPI: 'https://transaction-testnet.safe.xai.games',
```
And this one should be for prod Testnet service